src/components/views/project/projectActionCard/QFSection.tsx

```diff
@@ -90,9 +90,14 @@ const QFSection: FC<IQFSectionProps> = ({ projectData }) => {
 	};
 
 	const getEstimatedMatchingRange = (esMatching: number) => {
-		if (esMatching >= 1) {
+		if (esMatching >= 2) {
 			return `${formatWithCurrency((esMatching * 30) / 100, allocatedFundUSDPreferred ? '$' : '', locale)} - ${formatWithCurrency(esMatching, allocatedFundUSDPreferred ? '$' : '', locale)}`;
 		}
+
+		if (esMatching >= 1) {
+			return '< $2';
+		}
```

Comment:
This logic is used multiple times. Is it possible to use a reusable function instead of `getEstimatedMatchingRange`?